apps/webservice/src/app/[workspaceSlug]/settings/(settings)/account/profile/page.tsx

```diff
@@ -1,15 +1,142 @@
-export const metadata = { title: "Profile" };
+import React from "react";
+import Image from "next/image";
+import { redirect } from "next/navigation";
+import { SiGithub, SiGooglecalendar, SiSlack } from "react-icons/si";
+
+import { auth } from "@ctrlplane/auth";
+import { Button } from "@ctrlplane/ui/button";
+import { Input } from "@ctrlplane/ui/input";
+import { Label } from "@ctrlplane/ui/label";
+
+import { api } from "~/trpc/server";
+import UpsertUsername from "./UpsertUsername";
+
+const defaultAvatar = "/apple-touch-icon.png";
+
+export default async function AccountSettingProfilePage() {
+  const session = await auth();
```

Comment:
does session not have user info?

Reply:
It doesn't, just the id. 
```tsx
declare module "next-auth" {
  interface Session {
    token: JWT;
    user: {
      id: string;
    } & DefaultSession["user"];
  }
}
```

And DefaultSession only has:
```tsx
export interface User {
  id?: string
  name?: string | null
  email?: string | null
  image?: string | null
}
```